src/oop/object_independence/medium/q3/TestRecipe.java

```diff
@@ -0,0 +1,10 @@
+package oop.object_independence.medium.q3;
+
+public class TestRecipe {
+    public static void main(String[] args) {
```

Comment:
For this category, we want the students to modify the object to show that the two instantiated objects are independent of each other.